qcelemental/tests/test_molparse_reconcile_nucleus.py

```diff
@@ -0,0 +1,211 @@
+import pytest
+
+import qcelemental
+
+co_dominant = (59, 27, 'Co', 58.93319429, True, '')
+co_dominant_mine = (59, 27, 'Co', 58.93319429, True, '_mine')
+co_dominant_shortmass = (59, 27, 'Co', 58.933, True, '')
+co60 = (60, 27, 'Co', 59.93381630, True, '')
+co60ghost = (60, 27, 'Co', 59.93381630, False, '')
+co_unspecified = (-1, 27, 'Co', 60.6, True, '')
+
+
+def test_reconcile_nucleus_1():
```

Comment:
Great spot for parametrize. Also need to look into the float comparison using `==`.